openedx/core/djangoapps/user_authn/api/views.py

```diff
@@ -69,9 +69,10 @@ def get(self, request, **kwargs):  # lint-amnesty, pylint: disable=unused-argume
 
             optional_fields = RegistrationFieldsContext('optional').get_fields()
             if optional_fields:
+                extended_profile_fields = configuration_helpers.get_value('extended_profile_fields')
                 context['optional_fields'].update({
                     'fields': optional_fields,
-                    'extended_profile': configuration_helpers.get_value('extended_profile_fields', []),
```

Comment:
Theoretically, `configuration_helpers.get_value('extended_profile_fields', [])` should return `[]` if `extended_profile_fields` is not found. 
Why is this code not working as expected? can we do something to make it work instead of ternary operator?

Reply:
@syedsajjadkazmii the method `get_value` explicitly returns the dict. 
https://github.com/openedx/edx-platform/blob/cfc8ab629826cb407fb7189b61e3bed56a97a2b4/openedx/core/djangoapps/site_configuration/helpers.py#L143